types/react/index.d.ts

```diff
@@ -1737,9 +1741,17 @@ declare namespace React {
      * A {@link Dispatch} function can sometimes be called without any arguments.
      */
     type DispatchWithoutAction = () => void;
-    // Unlike redux, the actions _can_ be anything
+    /**
+     * A function to manage the state updates {@link useReducer} hook
```

Comment:
We don't use the term "state updates useReducer hook". Could you check what we use in the react.dev docs for naming the reducer function to match?